www/_posts/2021-07-30-cordova-android-10.0.1.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 10.0.1 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 10.0.1`!  This is one of Cordova's supported platforms for building Android mobile applications.
+
+* [cordova-android@10.0.1](https://www.npmjs.com/package/cordova-android)
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@10.0.1
+```
+
+## Release Highlights
+
+In this patch release, we had fixed three reported issues that caused build issues.
+
+* **Fixed `GradlePluginGoogleServicesEnabled` Invalid Version Error**
+
+Projects that set the `GradlePluginGoogleServicesEnabled` config flag to `true` saw an invalid version error. The error was caused by the build process reading a bad variable reference. 
+
+We corrected the variable reference and fixed this issue.
```

Comment:
I don't think we need to say how things were fixed, unless there are new instructions that end users need to be aware of